tasks/prime.py

```diff
@@ -17,49 +19,55 @@
     MTO_SERVICE_ITEM,
     PAYMENT_REQUEST,
 )
-
 from .base import check_response, CertTaskMixin, ParserTaskMixin
-from copy import deepcopy
 
 logger = logging.getLogger(__name__)
 
 
-def prime_path(url):
+def prime_path(url: str) -> str:
```

Comment:
Does this syntax just mean the url has to be a string? I don't thin I have seen it before.

Reply:
yep! This is called "type hinting" in Python, and it's a way we can try to make python a little stricter with data types. This line is saying that `url` needs to be a string (the `url: str` part), and it's also saying that this function will always return a string (the `-> str` part)

Reply:
Should we aim to do this in all of our functions?

Reply:
There's no hard and fast rule one way or the other - if you have a function that only works with clearly-defined types, it would be a good idea to use type-hinting. Still, there's a lot of grey area with types in python, and there are situations where it makes less sense. I actually like to take advantage of python's loose typing though, and I know some python developers who hate that haha. 

At the very least, I would encourage using type hinting if you can. It's rarely (if ever - arguably) a bad thing, and although I don't want to make it a hard requirement, it would be nice to see more of it.